ts/theme/variables.ts

```diff
@@ -97,7 +97,7 @@ const customVariables = Object.assign(materialVariables, {
   // Footer
   footerBackground: "#FFFFFF",
   footerElevation: 50,
-  footerPaddingTop: 24,
+  footerPaddingTop: 16,
```

Comment:
what impact does this have on other screens? - @fpersico

Reply:
Already discussed with @fgiobergia in pvt. It is ok.